src/fetchers.ts

```diff
@@ -298,28 +297,60 @@ async function fetchHelm(
 }
 
 async function fetchNixpkgs(
-  spec: NixpkgsFetch,
+  _spec: NixpkgsFetch,
   context: FetchContext,
 ): Promise<Version[]> {
-  const version = await fetchUrl("https://channels.nixos.org/" + spec.channel, context, undefined, { redirect: "manual" }, async (response) => {
-    const regexp = /releases.nixos.org\/nixpkgs\/([^/]+)/;
-    if (response.status !== 301) {
-      throw new Error(
-        `Got status ${response.status}: ${await response.text()}`,
-      );
-    }
-    const location = response.headers.get("Location");
-    if (location === null) {
-      throw new Error(`Could not get location for ${response.url}`);
-    }
-    const match = regexp.exec(location);
-    if (match === null) {
-      throw new Error(`Could not parse ${response.url}`);
+  const cacheKey = "nix-channel:nixpkgs-unstable";
```

Comment:
This doesn't use the standard url-based cache so I'm using the `nix-channel:` scheme here. We only ever check `nixpkgs-unstable` so I'm hardcoding it here but if we do add support for other channels then it's open to extension.